### DISCUSSION TITLE
wsl crashes after I used lxrunoffline  to move it from c drive

### DISCUSSION THREAD

**Discussion by dhirajnitk** [microsoft/WSL#6637] [Q&A]
I am unable to launch wsl after I used lxrunoffline   to move the filesystem from c to another drive.

**Comment by Biswa96** [comment]
This issue should be in LxRunOffline repository. To move WSL distribution there are some official docs.

**Reply by dhirajnitk** [reply]
`wsl --list --verbose`

> Ubuntu-20.04    Stopped         1

I can see the ubuntu distribution in the drive. wsl is able to recognise the ubuntu so its a terminal launch issue
Is there any mean to login the output of wsl?

**Reply by dhirajnitk** [reply]
I go to store and it shows ubuntu 20.04 is installed on the device. I try to launch and the launcher runs and goes off
